images/macos/provision/core/Configure-Toolset.ps1

```diff
@@ -0,0 +1,60 @@
+################################################################################
+##  File:  Configure-Toolset.ps1
+##  Team:  CI-Build
+##  Desc:  Configure toolset
+################################################################################
+
+Import-Module "~/image-generation/helpers/Common.Helpers.psm1"
+
+function Get-ToolsetToolFullPath
+{
+    param
+    (
+        [Parameter(Mandatory)] [string] $ToolName,
+        [Parameter(Mandatory)] [string] $ToolVersion,
+        [Parameter(Mandatory)] [string] $ToolArchitecture
+    )
+
+    $toolPath = Join-Path -Path $env:AGENT_TOOLSDIRECTORY -ChildPath $toolName
+    $toolPathVersion = Join-Path -Path $toolPath -ChildPath $toolVersion
+    $foundVersion = Get-Item $toolPathVersion | Sort-Object -Property {[version]$_.name} -Descending | Select-Object -First 1
+    $installationDir = Join-Path -Path $foundVersion -ChildPath $toolArchitecture
+    return $installationDir
+}
+
+function Add-EnvironmentVariable
```

Comment:
I think, we should move this function to the Common.Helpers.psm1. It can be reused in the [Xcode.Installer.psm1](https://github.com/actions/virtual-environments/blob/main/images/macos/helpers/Xcode.Installer.psm1#L217)

Reply:
Moved it